DataFormats/HGCDigi/src/classes.h

```diff
@@ -18,22 +24,6 @@ namespace DataFormats_HGCDigi {
     std::vector<HGCDataFrame<HcalDetId,HGCSample> > vHGCalBHDataFrames;
     edm::SortedCollection< HGCDataFrame<HcalDetId,HGCSample> > scHGCalBHDataFrames;
     edm::Wrapper< edm::SortedCollection< HGCDataFrame<HcalDetId,HGCSample> > > prodHGCalBHDataFrames;
-    
```

Comment:
@bsunanda @kpedro88 were these DataFormats already used in past productions?

Reply:
@fabiocos yes, that's why we kept them here.